torch/quantization/fake_quantize.py

```diff
@@ -212,8 +212,8 @@ def __init__(self,
         assert quant_min <= quant_max, 'quant_min should be less than or equal to quant_max'
         self.quant_min = quant_min
         self.quant_max = quant_max
-        self.register_buffer('scale', torch.tensor([scale]))
-        self.register_buffer('zero_point', torch.tensor([zero_point]))
+        self.register_buffer('scale', torch.tensor([scale], dtype=torch.float))
+        self.register_buffer('zero_point', torch.tensor([zero_point], dtype=torch.float))
```

Comment:
qq, what's the context for this one?

Reply:
added when debugging, just to make it more explicit, is this needed?

Reply:
makes sense, I was just wondering, lg

Reply:
update: removed for now, looks like we didn't do that in fake_quantize.py right now, maybe we can add this back when we have better numerical test coverage on the qat conversion later.